waspc/src/Wasp/Lib.hs

```diff
@@ -88,11 +92,17 @@ findWaspFile waspDir = do
       ".wasp" `isSuffixOf` SP.toFilePath path
         && (length (SP.toFilePath path) > length (".wasp" :: String))
 
-findDotEnvFile :: Path' Abs (Dir WaspProjectDir) -> IO (Maybe (Path' Abs File'))
-findDotEnvFile waspDir = do
-  let dotEnvAbsPath = waspDir SP.</> [relfile|.env|]
-  dotEnvExists <- doesFileExist (SP.toFilePath dotEnvAbsPath)
-  return $ if dotEnvExists then Just dotEnvAbsPath else Nothing
+findDotEnvServer :: Path' Abs (Dir WaspProjectDir) -> IO (Maybe (Path' Abs File'))
+findDotEnvServer waspDir = toMaybeAbsFile waspDir dotEnvServer
+
+findDotEnvClient :: Path' Abs (Dir WaspProjectDir) -> IO (Maybe (Path' Abs File'))
+findDotEnvClient waspDir = toMaybeAbsFile waspDir dotEnvClient
+
+toMaybeAbsFile :: Path' Abs (Dir WaspProjectDir) -> Path' (SP.Rel WaspProjectDir) File' -> IO (Maybe (Path' Abs File'))
+toMaybeAbsFile waspDir dotEnvFile = do
+  let dotEnvAbsFp = waspDir SP.</> dotEnvFile
+  dotEnvFileExists <- doesFileExist $ SP.toFilePath dotEnvAbsFp
+  return $ if dotEnvFileExists then Just dotEnvAbsFp else Nothing
```

Comment:
I would propose following refactoring:

```suggestion
findFileInWaspProjectDir
  :: Path' Abs (Dir WaspProjectDir)
  -> Path' (SP.Rel WaspProjectDir) File'
  -> IO (Maybe (Path' Abs File'))
findFileInWaspProjectDir waspDir file = do
  let fileAbsFp = waspDir SP.</> file
  fileExists <- doesFileExist $ SP.toFilePath fileAbsFp
  return $ if fileExists then Just fileAbsFp else Nothing
```

Basically two changes:
1. Renamed function to better reflect what it does.
2. Renamed its internals to make it aligned with the fact that it works for any file, not just env file.

Reply:
I agree. I also noticed that this general purpose function can be applied to any file, not just dot-env but was too shy to change proposed code.